webui/module/Auth/src/Auth/Controller/AuthController.php

```diff
@@ -130,11 +130,11 @@ public function loginAction()
       if(isset($bareos_updates) && $bareos_updates == false) {
          // updates could not be retrieved by ajax call
          $_SESSION['bareos']['product-updates-status'] = false;
-         return null;
```

Comment:
not returning here will make the rest of the code run. Line 176 uses `$updates` as an array without checking it before.
This branch should therefore initialize it with somewhat like `$updates = []`.

Reply:
true, made a few adjustments